packages/frontend/src/components/UserSettings/PasswordPanel/index.tsx

```diff
@@ -19,6 +29,10 @@ const PasswordPanel: FC = () => {
     const { isLoading, mutate: updateUserPassword } =
         useUserUpdatePasswordMutation();
 
+    const handleChange = (password: string) => {
```

Comment:
I think this may require an useCallback wrapper, but I think you can get rid of this method by doing `setValidationResult(validatePassword(password));` inside the onChange